_rules/image-no-text-0va7u6.md

```diff
@@ -36,12 +36,12 @@ This rule applies to any [rendered image resources][] in a [web page][].
 
 ## Expectation
 
-For each test target, at least one of the following is true:
+
+For each test target, its [visible pixels][visible] do not contain text expressing anything in a [human language], except if at least one of the following is true:
 
 - **essential**: the [visible pixels][visible] of the test target contain text for which its presentation is [essential][]; or
```

Comment:
I'm not so sure we should use "essential" here. It's sort of out of scope of this PR, but it's very relevant in moving this rule forward. We had a discussion about "essential" last week in ACT TF and our impression there was that it was too ambiguous to be used in an expectation.

Reply:
I'll open an issue. This is not related to your PR. Feel free to move forward with that.